conf/flash_modes.xml

```diff
@@ -91,6 +91,8 @@
       <board name="matek_.*"/>
       <board name="holybro_.*"/>
       <board name="cube_.*"/>
+      <board name="betafpv_aiof7"/>
+      <board name="tmotor_aiof7"/>
```

Comment:
Is `SWD` the only flashing mode available ?

Reply:
No, in usb dfu too! 
By default, the card is added to the paparazzi center in default mode.